src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -299,13 +301,17 @@ public FormValidation doCheckHookUrl(@QueryParameter String value) {
                     return FormValidation.warning("It doesn't look like " + value + " is talking to any Jenkins. Are you running your own app?");
                 }
                 RSAPublicKey key = identity.getPublic();
-                String expected = new String(Base64.encodeBase64(key.getEncoded()));
+                String expected = new String(Base64.encodeBase64(key.getEncoded()), "UTF-8" );
```

Comment:
it compiled successfully with such const. I've already use it in some places without changing target source and target lang level